lib/src/models/networks.dart

```diff
@@ -38,28 +40,34 @@ final bitcoin = new NetworkType(
     bip32: new Bip32Type(public: 0x0488b21e, private: 0x0488ade4),
     pubKeyHash: 0x00,
     scriptHash: 0x05,
-    wif: 0x80);
+    wif: 0x80,
+    opreturnSize: 80);
 
 final testnet = new NetworkType(
     messagePrefix: '\x18Bitcoin Signed Message:\n',
     bech32: 'tb',
     bip32: new Bip32Type(public: 0x043587cf, private: 0x04358394),
     pubKeyHash: 0x6f,
     scriptHash: 0xc4,
-    wif: 0xef);
+    wif: 0xef,
+    opreturnSize: 80);
 
-final peercoin = new NetworkType(
-    messagePrefix: '\x18Peercoin Signed Message:\n',
-    bech32: 'pc',
-    bip32: new Bip32Type(public: 0x0488b21e, private: 0x0488ade4),
-    pubKeyHash: 0x37,
-    scriptHash: 0x75,
-    wif: 0xb7);
+final peercoin = NetworkType(
+  messagePrefix: '\x18Peercoin Signed Message:\n',
+  bech32: 'pc',
+  bip32: Bip32Type(public: 0x043587cf, private: 0x04358394),
+  pubKeyHash: 0x37,
+  scriptHash: 0x75,
+  wif: 0xb7,
+  opreturnSize: 256,
+);
 
-final peercoinTestnet = new NetworkType(
-    messagePrefix: '\x18Peercoin Signed Message:\n',
-    bech32: 'tpc',
-    bip32: new Bip32Type(public: 0x043587cf, private: 0x04358394),
-    pubKeyHash: 0x6f,
-    scriptHash: 0xc4,
-    wif: 0xef);
+final peercoinTestnet = NetworkType(
+  messagePrefix: '\x18Peercoin Signed Message:\n',
+  bech32: 'tpc',
+  bip32: Bip32Type(public: 0x043587cf, private: 0x04358394),
+  pubKeyHash: 0x6f,
+  scriptHash: 0xc4,
+  wif: 0xef,
+  opreturnSize: 256,
+);
```

Comment:
I prefer the style of closing parentheses on a new line like this. However the dart formatting guidelines suggest that continuation lines should be indented by 4 spaces and the code example shows the closing parenthesis on the same line as the last parameter: https://github.com/dart-lang/dart_style/wiki/Formatting-Rules#indent-continued-lines-with-at-least-four-spaces

I notice elsewhere in the library that parentheses are not given on a new line. See how it is done for the `bitcoin` network: https://github.com/Vesta-wallet/coinslib/blob/eed5ee3c606bac687d8ea6e5537256421c675106/lib/src/models/networks.dart#L37

Should we stick to the old convention that is suggested for dart, or change to this new formatting?

Reply:
I don't really care about the trailing comma. I use it because I think it makes things look nicer. 

I don't use the "new" key word. In peercoin_flutter I use the "pedantic" package, which is the linter used by google devs... but that one would probably scream at this library.

Reply:
@bananenwilly I tried using the lints/recommended.yaml lints and got "432 issues found". It doesn't complain about putting the closing parentheses on a new line though. I agree that placing them on a new line is more readable. This:

```
aFunctionCall(
    parameter1,
    parameter2,
    named: parameter,
);
```

Is cleaner than:

```
aFunctionalCall(
    parameter1,
    parameter2,
    named: parameter);
```

But the library uses the latter style and sometimes (inconsistently) puts the first parameter on the same line as the function name.

For new code, I can follow the new style but we'd need to reformat the old code for it to be consistent. Maybe this is not a primary concern right now if we are going to rewrite a lot of the code eventually anyway.

Should I go ahead and follow the new style and create an issue to come back to this later?

Reply:
yes, new code with a) and the rest later 👍🏻

Reply:
Great. I'll create an issue to keep note of this.